data.sql

```diff
@@ -1,18 +1,78 @@
-	   ('Gabumon', '2018/11/15', 2, TRUE, 8.0),
-	   ('Pikachu', '2021/01/7', 1, FALSE, 15.04),
-	   ('Devimon', '2017/05/12', 5, TRUE, 11.0);
-
-	--    add these insertions to your data.sql file
-
-INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg)
-VALUES ('Charmander', '2020/02/08', 0, TRUE, -11.0), 
-	   ('Plantmon', '2021/11/15', 2, TRUE, -5.7),
-	   ('Squirtle', '1993/04/02', 3, FALSE, -12.13),
-	   ('Angemon', '2005/06/12', 1, TRUE, -45.0),
-	   ('Boarmon', '2005/06/07', 7, TRUE, 20.4),
-	   ('Blossom', '1998/10/13', 3, TRUE, 17.0),
-	   ('Ditto', '2022/05/14', 4, TRUE, 22.0);
\ No newline at end of file
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Agumon', '2020-02-03', 0, true, 10.23);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Gabumon', '2018-11-15', 2, true, 8);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Pikachu', '2021-01-07', 1, false, 15.04);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Devimon', '2017-05-12', 5, true, 11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Charmander', '2020-02-08', 0, false, -11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Plantmon', '2021-11-15', 2, true, -5.7);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Squirtle', '1993-04-02', 3, false, -12.13);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Angemon', '2005-06-12', 1, true, -45);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Boarmon', '2005-06-07', 7, true, 20.4);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Blossom', '1998-10-13', 3, true, 17);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Ditto', '2022-05-14', 4, true, 22);
+
+INSERT INTO owners
+(full_name, age)
+VALUES ('Sam Smith', 34), ('Jennifer Orwell', 19), ('Bob', 45),
+       ('Melody Pond', 77),('Dean Winchester', 14),('Jodie Whittaker', 38);
+
+INSERT INTO species
+(name)
+VALUES ('Pokemon'),('Digimon');
+
+UPDATE animals
+SET species_id = 2
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species_id = 1
+WHERE species_id IS NULL;
+
+BEGIN;
+
+SAVEPOINT Sam;
+UPDATE animals
+SET owner_id = 1
+WHERE name = 'Agumon';
+
+SAVEPOINT JenniferOrwell;
+UPDATE animals
+SET owner_id = 2
+WHERE name = 'Gabumon' OR name = 'Pikachu';
+
+SAVEPOINT Bob;
+UPDATE animals
+SET owner_id = 3
+WHERE name = 'Devimon' OR name = 'Plantmon';
+
+SAVEPOINT MelodyPond;
+UPDATE animals
+SET owner_id = 4
+WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';
+
+SAVEPOINT DeanWinchester;
+UPDATE animals
+SET owner_id = 5
```

Comment:
- [ ] As pointed out earlier, please make sure the `owner_id` is not hardcoded and gotten from the database instead.

Hint
`SET owner_id = (SELECT id from owners WHERE full_name = 'Sam Smith')`

Reply:
Thank you. I will quickly make the changes as required